packages/types/package.json

```diff
@@ -38,6 +38,7 @@
     "test:types": "tsd"
   },
   "dependencies": {
+    "@appium/logger": "^1.1.0",
```

Comment:
not sure if this is a good idea. We only need types there, which means it must eb only a dev dependency

Reply:
I agree that the whole logger functionality is of course not necessary here. Maybe it's possible to move the `Logger` type to `@appium/schema` and avoid this dependency?